Index: postcss.config.mjs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>const config = {\r\n  plugins: {\r\n    \"@tailwindcss/postcss\": {},\r\n  },\r\n};\r\n\r\nexport default config;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/postcss.config.mjs b/postcss.config.mjs
--- a/postcss.config.mjs	(revision 92fb1ef65ace4ce091592777c6750a5e17b01a3a)
+++ b/postcss.config.mjs	(date 1763384196562)
@@ -1,6 +1,7 @@
 const config = {
   plugins: {
-    "@tailwindcss/postcss": {},
+    '@tailwindcss/postcss': {},
+    autoprefixer: {},
   },
 };
 
Index: eslint.config.mjs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import { defineConfig, globalIgnores } from \"eslint/config\";\r\nimport nextVitals from \"eslint-config-next/core-web-vitals\";\r\nimport nextTs from \"eslint-config-next/typescript\";\r\n\r\nconst eslintConfig = defineConfig([\r\n  ...nextVitals,\r\n  ...nextTs,\r\n  // Override default ignores of eslint-config-next.\r\n  globalIgnores([\r\n    // Default ignores of eslint-config-next:\r\n    \".next/**\",\r\n    \"out/**\",\r\n    \"build/**\",\r\n    \"next-env.d.ts\",\r\n  ]),\r\n]);\r\n\r\nexport default eslintConfig;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/eslint.config.mjs b/eslint.config.mjs
--- a/eslint.config.mjs	(revision 92fb1ef65ace4ce091592777c6750a5e17b01a3a)
+++ b/eslint.config.mjs	(date 1763019890547)
@@ -1,3 +1,4 @@
+import prettierConfig from "eslint-config-prettier";
 import { defineConfig, globalIgnores } from "eslint/config";
 import nextVitals from "eslint-config-next/core-web-vitals";
 import nextTs from "eslint-config-next/typescript";
@@ -13,6 +14,7 @@
     "build/**",
     "next-env.d.ts",
   ]),
+    prettierConfig,
 ]);
 
 export default eslintConfig;
Index: tsconfig.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"compilerOptions\": {\r\n    \"target\": \"ES2017\",\r\n    \"lib\": [\"dom\", \"dom.iterable\", \"esnext\"],\r\n    \"allowJs\": true,\r\n    \"skipLibCheck\": true,\r\n    \"strict\": true,\r\n    \"noEmit\": true,\r\n    \"esModuleInterop\": true,\r\n    \"module\": \"esnext\",\r\n    \"moduleResolution\": \"bundler\",\r\n    \"resolveJsonModule\": true,\r\n    \"isolatedModules\": true,\r\n    \"jsx\": \"react-jsx\",\r\n    \"incremental\": true,\r\n    \"plugins\": [\r\n      {\r\n        \"name\": \"next\"\r\n      }\r\n    ],\r\n    \"paths\": {\r\n      \"@/*\": [\"./*\"]\r\n    }\r\n  },\r\n  \"include\": [\r\n    \"next-env.d.ts\",\r\n    \"**/*.ts\",\r\n    \"**/*.tsx\",\r\n    \".next/types/**/*.ts\",\r\n    \".next/dev/types/**/*.ts\",\r\n    \"**/*.mts\"\r\n  ],\r\n  \"exclude\": [\"node_modules\"]\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/tsconfig.json b/tsconfig.json
--- a/tsconfig.json	(revision 92fb1ef65ace4ce091592777c6750a5e17b01a3a)
+++ b/tsconfig.json	(date 1763024972670)
@@ -1,7 +1,11 @@
 {
   "compilerOptions": {
     "target": "ES2017",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
     "allowJs": true,
     "skipLibCheck": true,
     "strict": true,
@@ -19,7 +23,9 @@
       }
     ],
     "paths": {
-      "@/*": ["./*"]
+      "@/*": [
+        "./src/*"
+      ]
     }
   },
   "include": [
@@ -30,5 +36,7 @@
     ".next/dev/types/**/*.ts",
     "**/*.mts"
   ],
-  "exclude": ["node_modules"]
+  "exclude": [
+    "node_modules"
+  ]
 }
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># See https://help.github.com/articles/ignoring-files/ for more about ignoring files.\r\n\r\n# dependencies\r\n/node_modules\r\n/.pnp\r\n.pnp.*\r\n.yarn/*\r\n!.yarn/patches\r\n!.yarn/plugins\r\n!.yarn/releases\r\n!.yarn/versions\r\n\r\n# testing\r\n/coverage\r\n\r\n# next.js\r\n/.next/\r\n/out/\r\n\r\n# production\r\n/build\r\n\r\n# misc\r\n.DS_Store\r\n*.pem\r\n\r\n# debug\r\nnpm-debug.log*\r\nyarn-debug.log*\r\nyarn-error.log*\r\n.pnpm-debug.log*\r\n\r\n# env files (can opt-in for committing if needed)\r\n.env*\r\n\r\n# vercel\r\n.vercel\r\n\r\n# typescript\r\n*.tsbuildinfo\r\nnext-env.d.ts\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 92fb1ef65ace4ce091592777c6750a5e17b01a3a)
+++ b/.gitignore	(date 1763026005608)
@@ -31,7 +31,7 @@
 .pnpm-debug.log*
 
 # env files (can opt-in for committing if needed)
-.env*
+.env.local
 
 # vercel
 .vercel
@@ -39,3 +39,6 @@
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+# prisma
+prisma/schema.prisma
